.vib/ksql/goss/vars.yaml

```diff
@@ -0,0 +1,25 @@
+binaries:
+  - gosu
+  - java
+  - ksql
+  - ksql-datagen
+  - ksql-print-metrics
+  - ksql-run-class
+  - ksql-server-start
+  - ksql-server-stop
+  - ksql-stop
+  - ksql-test-runner
+  - ksql-restore-command-topic
```

Comment:
I'm not a fan of checking every single binary here, as we are introducing flakiness future-wise.